src/app/views/initial-setup/initial-setup-name/initial-setup-name.component.html

```diff
@@ -2,13 +2,11 @@
   <app-close-icon></app-close-icon>
   <div class="initial-setup-name__logo">
     <img alt="Green Traveller Logo" src="assets\icons\android-chrome-512x512.png" height="128" width="128">
+    <h3 class="initial-setup-name__headline">Green Traveller</h3>
   </div>
   <div class="initial-setup-name__text" id="textField" (click)="setFocusInput()">
-    <h4>
-      Hi  
-      <span class="initial-setup-name__input" id="inputName" contenteditable="true" autofocus>{{username}}</span>,
-    </h4>  
-    <h4>why don't you tell us your name and we personalize this process a little bit!</h4>
+    <h5>Great to see you here.</h5>
+    <h5>Lets start with your name: <span class="initial-setup-name__input" id="inputName" contenteditable="true" autofocus>{{username}}</span></h5> 
```

Comment:
I would move the username input in a new line

and then it would make sense to use a normal user input (<input type=text class=form-crontrol>) only that you will set the background color and text-align: center

Reply:
new try